__tests__/tasks/task-dev-dependency.test.js

```diff
@@ -0,0 +1,106 @@
+const API_BASE_URL = 'https://api.realdevsquad.com';
```

Comment:
I think we already have defined the based URL in the test config can you please check

Reply:
I have checked, I couldnt find can you tell me which file? 
Also if you check task-dependency.test.js also has const API_BASE_URL = 'https://api.realdevsquad.com'; let me know if it can be replaced